racket/collects/racket/private/prop-template-metafunction.rkt

```diff
@@ -0,0 +1,69 @@
+(module prop-template-metafunction '#%kernel
+  (#%provide (protect metafunction
```

Comment:
Note: I'm not sure if it is okay to protect these identifiers in a separate module (previously they were defined & protected in `template.rkt`).